site/en/docs/web-platform/origin-trial-troubleshooting/index.md

```diff
@@ -103,6 +103,9 @@ token is provided via an external script, not a meta tag or inline script</a></l
   <br>
   <input class="w-checkbox" type="checkbox" id="check-workers">
   <label for="check-workers" class="w-ml--l"><a href="#workers">Worker access is enabled</a></label>
+  <br>
+  <input class="w-checkbox" type="checkbox" id="token-before-access">
```

Comment:
It looks like the `token-before-access` id here conflicts with the same `token-before-access` id for the heading further down, so clicking on the link doesn't jump down to the section - just keeps you on this line. See:
https://deploy-preview-3093--developer-chrome-com.netlify.app/docs/web-platform/origin-trial-troubleshooting/#token-before-access

Reply:
Good catch! I forgot to add `check-` to the checkbox id.

Now fixed.